projects/plugins/boost/app/assets/src/js/features/boost-pricing-table/boost-pricing-table.tsx

```diff
@@ -1,37 +1,73 @@
-import { type PricingSchema, usePricing } from '$lib/stores/pricing';
+import { usePricing } from '$lib/stores/pricing';
+import { useProduct } from '$lib/stores/product';
 import {
 	Button,
 	Notice,
 	PricingTable,
 	PricingTableColumn,
 	PricingTableHeader,
+	PricingTableItem,
 	ProductPrice,
 } from '@automattic/jetpack-components';
-import { boostFeatureList } from './lib/features';
 import { __ } from '@wordpress/i18n';
 
 type BoostPricingTableProps = {
-	pricing: PricingSchema;
 	onPremiumCTA: () => void;
 	onFreeCTA: () => void;
 	chosenFreePlan: boolean;
 	chosenPaidPlan: boolean;
 };
 
+type BoostPricingTablesFeaturesListProps = {
+	features: {
+		included: boolean;
+		description?: string;
+		info?: {
+			title?: string;
+			content: string;
+			class?: string;
+		};
+	};
+};
+
+const BoostPricingTablesFeaturesList = ( { features }: BoostPricingTablesFeaturesListProps ) => {
```

Comment:
It looks like it should be a `feature` as it handles a single feature.

Reply:
Oops 😅 Thanks for catching that.

Updated in 4b5062f.